chart/pyroscope/values.yaml

```
@@ -68,7 +69,8 @@ podSecurityContext:
   fsGroup: 101
 
 # -- Deployment securityContext
-securityContext: {}
+securityContext:
+  {}
```

If there is no strong reason to change the format, let's adhere widely spread convention:
```suggestion
securityContext: {}
```

---

Hi @kolesnikovae 
Thanks, I have reverted back those changes.